# stereoStream.msg
uint64 stereo_timestamp
float64 x_stereo
float64 y_stereo
float64 z_stereo

